SimPEG/Optimization.py

```diff
@@ -990,4 +990,18 @@ def findSearchDirection(self):
                     cgFlag = 1
                 # End CG Iterations
 
+            # Take a gradient step on the active cells if exist
+            if temp != self.xc.size:
+
+                rhs_a = (Active) * -self.g
+
+                dm_i = max( abs( delx ) )
+                dm_a = max( abs(rhs_a) )
+
+                delx = delx + rhs_a * dm_i / dm_a /10.
```

Comment:
is that potentially quite large? 

We might also want to make it a key word argument so that it can be tuned easily if necessary